Advance-1/oop.php

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=, initial-scale=1.0">
+  <title>Advance-1 OOP</title>
+  <style>
+    <?php include 'style.css'; ?>
+  </style>
```

Comment:
Please use proper css linking for attaching css file.Inclusion of php tag is not required here.

Reply:
I think it is required.